hostname.go

```diff
@@ -0,0 +1,19 @@
+package unilog
+
+import (
+	"fmt"
+	"regexp"
+)
+
+var hostnameRegex = regexp.MustCompile(`^([A-Za-z]+)`)
+
+// ParseHostname parses a hostname and returns the
+// host type (e.g. mybox1 and mybox-123.local will both
+// return 'mybox').
+func ParseHostname(hostname string) (string, error) {
```

Comment:
unilog is open-source and not necessarily Stripe specific; would making the tag an argument and plumbing it through internally make more sense? It'd also avoid parsing hostnames in random application code, which is a desirable property to reduce coupling to our infrastructure.